projects/vdk-core/src/vdk/internal/builtin_plugins/job_properties/datajobs_service_properties.py

```diff
@@ -78,6 +78,8 @@ def set_all_properties(self, properties: Dict[str, PropertyValue]) -> None:
             )  # keeps the outer scope originally-referenced dictionary intact
             for client in self._write_preprocessors:
                 try:
+                    if (properties.get("cipher_key")) is not None:
```

Comment:
This prevents the key from getting passed to the backend service. It won't get persisted at all. I don't think this is what we want here.

Reply:
Agreed.